docs/core/diagnostics/dotnet-trace.md

```diff
@@ -405,7 +405,7 @@ However, when you want to gain a finer control over the lifetime of the app bein
 
 ## View the trace captured from dotnet-trace
 
-On Windows, *.nettrace* files can be viewed on [PerfView](https://github.com/microsoft/perfview) for analysis: For traces collected on other platforms, the trace file can be moved to a Windows machine to be viewed on PerfView.
+On Windows, *.nettrace* files can be viewed on [Visual Studio](https://docs.microsoft.com/en-us/visualstudio/profiling/beginners-guide-to-performance-profiling?view=vs-2022#step-2-analyze-cpu-usage-data) or [PerfView](https://github.com/microsoft/perfview) for analysis: For traces collected on other platforms, the trace file can be moved to a Windows machine to be viewed on Visual Studio or PerfView.
```

Comment:
```suggestion
On Windows, you can view *.nettrace* files in [Visual Studio](/visualstudio/profiling/beginners-guide-to-performance-profiling#step-2-analyze-cpu-usage-data) or [PerfView](https://github.com/microsoft/perfview) for analysis.
```

Reply:
Thank you for your suggestions! I think it is important to leave in the last sentence for traces collected on other platforms.